### PR TITLE
CreateIncIssue command

### PR DESCRIPTION
**Pull request by osipex**
This PR decouples CreateIssue and CreateIncIssue (which is a custom type of the same) to use their own structs.
This prevents further code support problems.

### CODE REVIEW

**Review by songupta7 — APPROVED** [review]

File: command/create.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 	"regexp"
 )
 
-// Input struct presents input options for flyte command
+// Input struct presents input options for CreateIssue flyte command
```
Comment (pamelin) [inline command/create.go:28]: I would say don't add comments to the source code unless they re trying to explain something really super important and very complex.

I guess what might be more appropriate in this case is to rename `Input` -> `CreateIssueInput` or if you want to be super descriptive `CreateIssueCmdInput`

File: command/create.go [hunk 2]
```diff
@@ -37,6 +37,17 @@ type Input struct {
 	Reporter    string   `json:"reporter"`
 }
 
+// IncIssueInput struct presents input options for CreateIncIssue flyte command.
+// This is a custom command which will have it's own flows used by various team.
```
Comment (pamelin) [inline command/create.go:41]: Remove comments

File: command/create.go [hunk 3]
```diff
@@ -37,6 +37,17 @@ type Input struct {
...
+type IncIssueInput struct {
+	Project     string   `json:"project"`
+	IssueType   string   `json:"issuetype"`
+	Summary     string   `json:"summary"`
+	Description string   `json:"description"`
+	Labels      []string `json:"labels"`
+	Inc         string   `json:"incident"` // ServiceNow incident
+}
+
 // CreateIssueCommand is a default command to create issue with minimum parameters
```
Comment (pamelin) [inline command/create.go:51]: remove comments

**Comment by pamelin** [comment]
Don't add comments in the code try to refactor and explain yourself in code. And don't comment out code just delete it.

Beside I don't know what this is about in terms of requirements and business logic so for me it is hard to asses if this meets business requirements and this is what code review should be about. I would say get someone who understands what you are doing to review it, run, test it (I'm not sure if there are any)

File: command/create.go (hunk 3 above)
Comment (osipex) [reply command/create.go]: fixed

File: command/create.go (hunk 2 above)
Comment (osipex) [reply command/create.go]: Fixed

File: command/create.go (hunk 1 above)
Comment (osipex) [reply command/create.go]: Fixed

**Review by songupta7 — APPROVED** [review]

**Review by rneumaier — APPROVED** [review]

**Pull request merged**